index.html

```diff
@@ -10,188 +10,223 @@
     </div>
+
+
+    
     <section class="recnt-projects">
-        <div class="project-setion">
-            <i class="fa fa-arrow-down work-icon"></i>
-            <h6 class="work-title"><a href="#">Work</a></h6>
-        </div>
-        <h1 class="recent-project-title">MY RECENT WORK</h1>
+        <h2 class="recent-project-title">MY RECENT WORK</h2>
         <hr class="hrbar">
         <div class="projs grid-container">
             <div class="proj-1">
                 <div class="proj-img">
                     <img src="images/proj-img.png" alt="Project Screen Shot">
                 </div>
-                <div class="proj-info">
-                    <div class="proj-title">
-                        <h3>Multi-Post Stories</h3>
-                </div>
-                <div class="proj-desc">
-                    <p class="project-desc-p">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required. has been the industry's standard dummy text ever
-                        since the 1500s, when an unknown printer took a standard dummy text.
-                      </p>
-                </div>
-                </div>
-                <div class="">
-                    <ul class="proj-tech">
-                        <li class="border-class">css</li>
-                        <li class="border-class">html</li>
-                        <li class="border-class">bootstrap</li>
-                        <li class="noborder">ruby</li>
-                    </ul>  
-                </div>
-                <div class="proj-btn">
-                    <button type="button" class="btn btn-primary">See Project</button>
+                <div class="proj-det">
+                    <div class="proj-info">
+                        <div class="proj-title">
+                            <h3>Multi-Post Stories</h3>
+                    </div>
+                    <div class="proj-desc">
+                        <p class="project-desc-p">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. has been the industry's standard dummy text ever
+                            since the 1500s, when an unknown printer took a standard dummy text.
+                          </p>
+                    </div>
+                    </div>
+                    <div class="">
+                        <ul class="proj-tech">
+                            <li class="border-class">css</li>
+                            <li class="border-class">html</li>
+                            <li class="border-class">bootstrap</li>
+                            <li class="noborder">ruby</li>
+                        </ul>  
+                    </div>
+                    <div class="proj-btn">
+                        <button type="button" class="btn btn-primary">See Project</button>
+                    </div>
                 </div>
```

Comment:
- [ ] [OPTIONAL] I think it will be nice if you could please give some margin to the projects and increase the size of your project blank card. Your projects section are a bit stretched as compared to the Figma design. 

 Your Design           |
:------------------------:|
![e4](https://user-images.githubusercontent.com/39523312/149410658-7ea7c22a-4f08-4cd5-95e3-2101c1f48a58.png)

 Figma Design           |
:------------------------:|
![e3](https://user-images.githubusercontent.com/39523312/149410670-297ddd46-ff77-4324-a518-51cab35a89a1.png)